packages/nimble-blazor/Examples/Demo.Shared/Pages/ComponentsDemo.razor

```diff
@@ -59,6 +59,19 @@
                 Icon Toggle Button
             </NimbleToggleButton>
         </div>
+        <div class="sub-container">
+            <div class="container-label">Card</div>
+            <NimbleCard>
+                <span slot="title">Title of the card</span>
+                <nimble-number-field>Numeric field 1</nimble-number-field>
```

Comment:
The Nimble elements in the card content should be using the Blazor form of the element names, i.e. `NimbleNumberField`, `NimbleSelect`, `NimbleListOption` (with `Value`)

Reply:
Ah, of course, that makes sense. Fixed!